### PR TITLE
showing how to shellout in docs

### PR DESCRIPTION
**Pull request by rshade**

### CODE REVIEW

File: docs/ruby_usage.md [hunk 1]
```diff
@@ -152,3 +152,31 @@ describe command('ls -al /') do
   its('exit_status') { should eq 0 }
 end
 ```
+
+## Shelling out in tests
+
+When writing tests you can not use standard ruby methods to shellout as it tries to execute those commands locally.
+However, the `command` resource has a `.stdout` method that will allow you to manipulate the results.
+Using the above example, you could check the writes on several subdirectories.
```
Comment (chris-rock) [inline docs/ruby_usage.md:160]: Maybe we should add an example for a test file, so that users can copy paste.

**Review by chris-rock — COMMENTED** [review]
Great work @rshade This helps to make it more clear.

**Review by chris-rock — APPROVED** [review]
Looks like a great addition to me

**Review by adamleff — APPROVED** [review]
LGTM! 👍 Thanks for your contribution @rshade!

**Pull request merged**